src/Microsoft.AspNet.Authentication.Google/GoogleAuthenticationMiddleware.cs

```diff
@@ -33,23 +33,7 @@ public class GoogleAuthenticationMiddleware : OAuthAuthenticationMiddleware<Goog
             [NotNull] IOptions<ExternalAuthenticationOptions> externalOptions,
             [NotNull] IOptions<GoogleAuthenticationOptions> options,
             ConfigureOptions<GoogleAuthenticationOptions> configureOptions = null)
-            : base(next, dataProtectionProvider, loggerFactory, encoder, externalOptions, options, configureOptions)
-        {
-            if (Options.Notifications == null)
-            {
-                Options.Notifications = new GoogleAuthenticationNotifications();
-            }
-
-            if (Options.Scope.Count == 0)
```

Comment:
Yeah makes more sense here, as you don't want the options to have defaults, only if the user doesn't specify any do you then want to populate default scopes